src/olympia/files/models.py

```diff
@@ -706,6 +712,50 @@ class Meta:
         db_table = 'validation_annotations'
 
 
+class WebextPermission(ModelBase):
```

Comment:
It feels over-engineered to have a whole model for each permission, and then a number of many2many rows for each add-on. Do we actually need that granularity ? It's going to cost us each time we want to display the list. I would have just stored a `JSONField` with the permission names per `File` and that's it (either in `File` directly or in a separate model with a `OneToOneField` to `File`) ...